conf/code-lists/pointer-mapping.csv

```diff
@@ -68,6 +68,7 @@
 42A.67A.68A.$1.02A.$2.D028,declaration.items.$1.additionalDocument.$2.documentStatusReason
 42A.67A.68A.$1.02A.$2.17B.R058,declaration.items.$1.additionalDocument.$2.issuingAuthorityName
 42A.67A.68A.$1.02A.$2.22C.502,declaration.items.$1.additionalDocument.$2.documentWriteOff.documentQuantity
+42A.67A.68A.03A.226,
```

Comment:
What is this? Has no value part and is not involved in removal of reverse mapping code :thinking:

Reply:
Rebased off the wrong branch, good spot thanks